MedtrumKitUI/Views/Onboarding/PatchPrimingView.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  PatchPrimingView.swift
+//  MedtrumKit
+//
+//  Created by Bastiaan Verhaar on 05/04/2025.
+//
+
+import SwiftUI
+import LoopKitUI
+
+struct PatchPrimingView: View {
+    @ObservedObject var viewModel: PatchPrimingViewModel
+    
+    var body: some View {
+        VStack {
+            List {
+                Section {
+                    supportImage("fill_reservoir")
+                    VStack(alignment: .leading) {
+                        Text(LocalizedString("Connect your pump base to the patch, remove the residual air, and fill with insulin.", comment: "Label for prime steps"))
+                            .foregroundStyle(.primary)
+                        Text(LocalizedString("NOTE: A minimum of 70U is required for activation", comment: "Label for minimum requirements"))
```

Comment:
a requested feature in AAPS was to show the actual reservoir in this step, up to you if you want that though ;)